sysvinit/openbsd/cloudfinal.tmpl

```diff
@@ -0,0 +1,34 @@
+## template:jinja
+#!/bin/ksh
+
+# PROVIDE: cloudfinal
+# REQUIRE: LOGIN cloudconfig
+
+daemon="cloud-init"
+daemon_execdir="{{prefix}}/bin"
+daemon_flags="modules --mode final"
+daemon_rtable=0
+daemon_timeout=1
+daemon_user=root
+
+. /etc/rc.d/rc.subr
+
+rc_bg="YES" # (undefined or "YES")
+rc_usercheck="YES" # (undefined or "NO")
+
+rc_start() {
+    test -e /etc/cloud/cloud-init.disabled \
```

Comment:
please replace this `test && …` with a simple `if` like the FreeBSD rc files have.
an `if` is easier to follow, less easy to mess up.
We'll need to do that in the NetBSD rc files too.